extensions/amp-recaptcha-input/0.1/amp-recaptcha-service.js

```diff
@@ -157,7 +161,7 @@ export class AmpRecaptchaService {
           dev().assertElement(this.iframe_),
           'amp-recaptcha-action',
           message,
-          '*',
+          devAssert(this.recaptchaFrameOrigin_),
```

Comment:
can there be a race condition? `this.recaptchaFrameOrigin_` is initialize too late.

Reply:
Nope, because this is wrapped in `this.recaptchaApiReady_.promise.then(() => {`, which only resolves once the iframe is created, and responded with the `ready` message. The Origin is stored when the iframe is first created, thus this would have to be set. Asserting here just to hopefully catch bugs in the future.

Reply:
that sounds good